1225-my_institution/institution_administrator_edits_my_institution.feature

```diff
@@ -21,5 +22,19 @@ Feature: Institution Administrator edits My Institution
             | Display name       |
             | Short display name |
             | Archive name       |
+            | Vocabularies       |
         And they click Save
         Then they see a Notification that the changes were saved
+
+    @xxx
+    Scenario Outline: Institution Administrator sees fields for Vocabularies
+        Given that the user is logged in as Institution Administrator
+        And they click the menu item My Institution
+        And they see the My Institution page
+        When they see fields for "<Vocabulary>"
```

Comment:
```suggestion
        And they see the Vocabularies settings
        When they see fields for "<Vocabulary>"
```

Reply:
beskriver ikke de to setningene akkurat det samme med den endringen?

Reply:
tja, jo, kanskje. men jeg synes vi burde få med at dette er en innstilling som påvirker noe. Føler `Vocabularies` sier litt for lite. Hva med å endre til `Vocabulary settings` i 360 og 361 da?

Reply:
ja, det var lurt!